ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift

```diff
@@ -134,12 +132,22 @@ struct FocusSystemSettingsView: View {
             shouldDismiss = true
           },
           label: {
-            Text(Strings.FocusOnboarding.laterActionButtonTitle)
-              .font(.subheadline.weight(.semibold))
-              .foregroundColor(Color(braveSystemName: .textSecondary))
-              .dynamicTypeSize(dynamicTypeRange)
+            Text(
+              "\(Strings.FocusOnboarding.startBrowseActionButtonTitle) \(Image(systemName: "arrow.right"))"
+            )
+            .font(.subheadline.weight(.semibold))
+            .foregroundColor(Color(braveSystemName: .textInteractive))
+            .dynamicTypeSize(dynamicTypeRange)
+            .padding()
+            .frame(maxWidth: .infinity)
           }
         )
+        .clipShape(RoundedRectangle(cornerRadius: 12.0, style: .continuous))
+        .overlay(
+          RoundedRectangle(cornerRadius: 12.0)
+            .inset(by: 0.5)
+            .stroke(Color(braveSystemName: .dividerInteractive).opacity(0.6), lineWidth: 1)
```

Comment:
strokeBorder

Reply:
https://github.com/brave/brave-core/pull/23471/commits/278fd54efc2b46571485d77692db3b63d26df0ec

Reply:
Still needs a fix, you don't need the `inset(by: 0.5)` when you use `strokeBorder`

Reply:
Oh right, thanks for checking, fix https://github.com/brave/brave-core/pull/23471/commits/75e5ec81660e8c455fec514a55b07ae1d6b7a574